### PR TITLE
fix render error with reference not found

### PR DESCRIPTION
**Pull request by andrewroth**

### CODE REVIEW

File: app/views/fe/reference_sheets/not_found.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<%= _("We weren"t able to find a reference associated with this link. The are two common reason for this issue:") %>
+<%= _("We weren't able to find a reference associated with this link. The are two common reason for this issue:") %>
```
Comment (knutsenm) [inline app/views/fe/reference_sheets/not_found.html.erb:1]:
```suggestion
<%= _("We weren't able to find a reference associated with this link. There are two common reasons for this issue:") %>
```

**Review by knutsenm — CHANGES_REQUESTED** [review]

**Comment by knutsenm** [comment]
I also see problems in these files:

app/views/fe/admin/panels/_prop_choice_field.html.erb:47
app/views/fe/references/edit.html.erb:1
app/views/fe/submit_pages/edit.html.erb:17

**Comment by knutsenm** [comment]
And these:

app/views/fe/admin/panels/_prop_choice_field.html.erb:23 (note, different line number)
app/views/fe/admin/question_pages/_question_page.html.erb:11
app/views/fe/answer_sheets/_answer_sheet.html.erb:23
app/views/fe/questions/fe/_date_field_mmyy.html.erb:9

(found by looking for an odd number of `"` on a line)

**Comment by andrewroth** [comment]
Ah, good idea! I was trying with grep to pro-actively find some, and didn't have much luck. Odd number I hadn't thought about

**Comment by knutsenm** [comment]
Also,

app/views/fe/admin/panels/_prop_reference_question.html.erb:3,4
app/views/fe/reference_pages/_reference.html.erb:27

File: app/views/fe/reference_sheets/not_found.html.erb (hunk 1 above)
Comment (knutsenm) [reply app/views/fe/reference_sheets/not_found.html.erb:1]: Bump

**Comment by knutsenm** [comment]
> Also,
> 
> app/views/fe/admin/panels/_prop_reference_question.html.erb:3,4 app/views/fe/reference_pages/_reference.html.erb:27

Bump

**Comment by andrewroth** [comment]
I think I've got them all

**Comment by knutsenm** [comment]
> I think I've got them all

> > Also,
> >
> > app/views/fe/admin/panels/_prop_reference_question.html.erb:3,4
> > app/views/fe/reference_pages/_reference.html.erb:27
> Bump

Bump bump ;-)

**Review by knutsenm — APPROVED** [review]
🎉

**Pull request merged**